packages/discord-api/src/client.ts

```diff
@@ -0,0 +1,16 @@
+import pkg from "../package.json" assert { type: "json" };
+import Client from "discord-request";
+
+class DiscordApiClient extends Client {
+	setUserAgent(value: string) {
+		return super.setUserAgent(`${value}, discord-api ${pkg.version}`);
+	}
+}
+const instance = new DiscordApiClient({
```

Comment:
Double check these defaults and make sure there are ways of changing sweep intervals